getting_started/index.adoc

```diff
@@ -8,8 +8,33 @@
 :toc: macro
 :toc-title:
 
+[[introduction]]
+== Introduction
+{product-title} is a platform as a service (PaaS) offering from Red Hat
+that brings together Docker, Kubernetes, and an API to manage these services.
```

Comment:
The API manages docker and kube? If so, maybe: brings together Docker and Kubernetes, and provides and API to manage these services.